src/arch/riscv/RiscvISA.py

```diff
@@ -106,7 +107,9 @@ class RiscvISA(BaseISA):
         ELEN in Ch. 2 of RISC-V vector spec",
     )
     privilege_mode_set = Param.PrivilegeModeSet(
-        "MSU",
+        "MHSU",  # MHSU to enable hypervisor (H-extension)
```

Comment:
The MHSU mode should not be the default if the timing cpu/timing mode is not supported.

Reply:
Yeah that is totally reasonable, I just wanted to see how many of the tests were passing with H on. If H-extension is disabled a lot of logic is guarded out. 

I am going to set the default to MSU and push, hopefully tests will pass :)